docs/rfc/rfc-1.adoc

```diff
@@ -194,6 +194,16 @@ Governable Parameters
 - `sweep_max_btc`: The amount of total bitcoin that will trigger an early sweep.
 - `dust_threshold`: The minimum bitcoin deposit amount for the transaction to
   be included in a sweep.
+- `sweeping_fee_bump_period`: The amount of time we wait to see if a sweeping
+  tranaction is mined before increasing the fee.
+- `sweeping_fee_multiplier_increment`: The amount we add to the sweeping fee
+  multiplier each time a sweeping transaction is not mined within the
+  `sweeping_fee_bump_period`. For example, if this param is set to 0.2 and we
+  are currently at 1.6x, then the next time we would try 1.8x.
+- `sweeping_fee_max_multiplier`: The highest we will try to increment the fee
+  multiplier to before giving up and picking a new base fee and different
+  deposits to sweep.
+
 
 The operators sign a transaction that unlocks all revealed deposits, combines
```

Comment:
Maybe this PR would be good to update this sentence (see my comment https://github.com/keep-network/tbtc-v2/pull/62#discussion_r748040498)? We could simply mention here that only some of the deposits will be combined and link to `bitcoin-sweeping-fee` section for more info.

Reply:
addressed in 6b3d7d4